### PR TITLE
feat: add accepts parameter to examples [MONET-1826]

### PR DESCRIPTION
**Pull request by cemreyuksel**
## Purpose

[Ticket](https://contentful.atlassian.net/browse/MONET-1826)

## Approach

This PR adds accepts parameter to examples and to function templates.

### CODE REVIEW

File: examples/function-potterdb/build-functions.js [hunk 1]
```diff
@@ -32,9 +32,14 @@ const validateFunctions = () => {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
 
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate entryFile path: '${entryFile}'`);
```
Comment (mrlaessig) [inline examples/function-potterdb/build-functions.js:36]:
```suggestion
      throw new Error(`Duplicate accepts path: '${accepts}'`);
```

Copy&paste?

File: examples/function-templates/javascript/build-functions.js [hunk 2]
```diff
@@ -32,9 +32,14 @@ const validateFunctions = () => {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
 
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate entryFile path: '${entryFile}'`);
```
Comment (mrlaessig) [inline examples/function-templates/javascript/build-functions.js:36]: Copy&paste error message?

File: examples/function-templates/typescript/build-functions.js [hunk 3]
```diff
@@ -32,9 +32,14 @@ const validateFunctions = () => {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
 
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate entryFile path: '${entryFile}'`);
```
Comment (mrlaessig) [inline examples/function-templates/typescript/build-functions.js:36]: Same Copy&paste?

File: examples/function-potterdb/build-functions.js (hunk 1 above)
Comment (cemreyuksel) [reply examples/function-potterdb/build-functions.js:36]: Exactly, sorry :)

File: examples/function-templates/javascript/build-functions.js (hunk 2 above)
Comment (cemreyuksel) [reply examples/function-templates/javascript/build-functions.js:36]: All over same copy pasta :D

**Review by mrlaessig — APPROVED** [review]

File: examples/function-templates/typescript/build-functions.js [hunk 4]
```diff
@@ -31,10 +31,14 @@ const validateFunctions = () => {
     if (uniqueValues.has(entryFile)) {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate accepts path: '${entryFile}'`);
```
Comment (mrlaessig) [inline examples/function-templates/typescript/build-functions.js:35]:
```suggestion
      throw new Error(`Duplicate accepts path: '${accepts}'`);
```
?

File: examples/function-templates/javascript/build-functions.js [hunk 5]
```diff
@@ -31,10 +31,14 @@ const validateFunctions = () => {
     if (uniqueValues.has(entryFile)) {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate accepts path: '${entryFile}'`);
```
Comment (mrlaessig) [inline examples/function-templates/javascript/build-functions.js:35]:
```suggestion
      throw new Error(`Duplicate accepts path: '${accepts}'`);
```

?

File: examples/function-potterdb/build-functions.js [hunk 6]
```diff
@@ -31,10 +31,14 @@ const validateFunctions = () => {
     if (uniqueValues.has(entryFile)) {
       throw new Error(`Duplicate entryFile path: '${entryFile}'`);
     }
+    if (uniqueValues.has(accepts)) {
+      throw new Error(`Duplicate accepts path: '${accepts}'`);
```
Comment (mrlaessig) [inline examples/function-potterdb/build-functions.js:35]:
```suggestion
      throw new Error(`Duplicate accepts: '${accepts}'`);
```

**Review by t-col — APPROVED** [review]

**Pull request merged**